tests/test_diffraction_objects.py

```diff
@@ -164,11 +164,11 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
 
 
 def test_on_xtype():
```

Comment:
I think we could use `@pytest.mark.parametrize` for this test function?

Reply:
Yes, let's refactor

Reply:
Actually, this could be a good use of standard DOs in contftest if we also have the properly computed arrays that we can check against with well thought it names.

Reply:
E.g, sthg like `test_do` that shortens to to and instead of 2*np.pi we have`tdo_lambda` and so on. We just want it more readable everywhere so the intent comes through better and it is easier to maintain

Reply:
https://github.com/diffpy/diffpy.utils/issues/229

Yup, created an issue. I will handle this.